src/CitMovie.Api/Controller/BookmarkController.cs

```diff
@@ -34,8 +34,8 @@ public async Task<IActionResult> CreateBookmark(int userId, [FromBody] CreateBoo
             return BadRequest("Bookmark data is required.");
 
         createBookmarkDto.UserId = userId;
-        var createdBookmark = await _bookmarkManager.CreateBookmarkAsync(createBookmarkDto);
-        return CreatedAtAction(nameof(GetBookmark), new { userId, id = createdBookmark.BookmarkId }, createdBookmark);
```

Comment:
Why was this changed? The endpoint should return `created` status code not `ok`?

Reply:
CreatedAtAction was causing me some errors so i went for just returning a status code 200 with ok.
I can use return Created("Bookmark created successfully.", createBookmarkDto); which will return the 201 if that is important.

Reply:
I think we should use the correct status code but `Created is fine if you cant get `CreatedAtAction` to work

Reply:
Created will still return 201 to my understanding, which is the same CreatedAtAction will. 

See this:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.controllerbase?view=aspnetcore-8.0